desktop/core/src/desktop/js/apps/storageBrowser/StorageFilePage/StorageFilePage.test.tsx

```diff
@@ -305,7 +292,12 @@ describe('StorageFilePage', () => {
     expect(video.children[0]).toHaveAttribute('src', expect.stringContaining('videofile.mp4'));
   });
 
-  it('should display a message for compressed file types', () => {
+  it('should display a message for unsupported file types', () => {
+    mockData = {
+      isContentReadable: false,
+      contents: ''
+    };
```

Comment:
This could potentially change the outcome of other tests relying on mockData. Perhaps resetting it in an "beforeEach"..